packages/api/src/routes/taskRoute.js

```diff
@@ -38,6 +38,13 @@ router.patch(
   taskController.assignAllTasksOnDate(),
 );
 
+router.patch(
+  '/patch_due_date/:task_id',
+  hasFarmAccess({ params: 'task_id' }),
+  checkScope(['edit:task']),
+  taskController.patchTaskDate(),
```

Comment:
The parameters to an Express route handler like `patch` are a route string followed by a sequence of function names or definitions (not function calls); those functions will be called later when an incoming request matches the method and route. `hasFarmAccess` and `checkScope` are being called here because they return the function definitions that are passed to `patch` so that they can be called later.  hasFarmAccess and checkScope are examples of what Express calls "configurable middleware"-- they return a middleware function that is configured by the parameters passed here. `patchTaskDate` takes no parameters, which means that it always returns the same function. Since nothing is being configured, the norm with Express is to simply name the function, not call it, and simplify `patchTaskDate` so that it performs the job its name suggests instead of returning a function that performs that job.